nmtwizard/preprocess/consumer.py

```diff
@@ -137,10 +141,25 @@ def _log_info(file_info, file_summary, proc, units):
             summary = self._summary[proc]
             if summary:
                 all_files_summary = summary.pop(None)
-                _log_info("for the whole sample ", all_files_summary, proc, units)
+                total_sentence_number = sum(
+                    [v.get("linesampled") for v in self._sampler_summary.values()]
```

Comment:
No need to build the list to compute the sum:

```suggestion
                    v.get("linesampled") for v in self._sampler_summary.values()
```